TypeCobol.LanguageServer/Workspace.cs

```diff
@@ -53,8 +53,16 @@ public class Workspace
         public EventHandler<LoadingIssueEvent> LoadingIssueEvent { get; set; }
         public EventHandler<ThreadExceptionEventArgs> ExceptionTriggered { get; set; }
         public EventHandler<string> WarningTrigger { get; set; }
+        /// <summary>
+        /// Event When the client configuration has changed in reception of a DidChangeConfiguration notification
+        /// </summary>
+        public EventHandler<IEnumerable<string>> ClientConfigurationChangedEvent { get; set; }
         public Queue<MessageActionWrapper> MessagesActionsQueue { get; private set; }
         private Func<string, Uri, bool> _Logger;
+        /// <summary>
+        /// The Analyzer Provider.
+        /// </summary>
+        public AnalyzerProvider AnalyzerProvider { get; private set; }
```

Comment:
No need for a new property, you already have access to the `IAnalyzerProvider` of the current `CompilationProject`.

Reply:
Ok to remove it.